spikeextractors/extractors/yassextractors/yassextractors.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+import yaml
+import os
+
+from spikeextractors import SortingExtractor
+#from spikeextractors.extractors.numpyextractors import NumpyRecordingExtractor
+#from spikeextractors.extraction_tools import check_valid_unit_id
+
+try:
+    HAVE_SCSX = True
```

Comment:
no import here ?

Reply:
I dont' understand

Reply:
I will just assume YASS is installed then.